openam-oauth2/src/main/java/org/forgerock/oauth2/core/AuthorizeRequestValidatorImpl.java

```diff
@@ -13,11 +13,16 @@
  *
  * Copyright 2014-2016 ForgeRock AS.
  * Portions Copyrighted 2018 Open Source Solution Technology Corporation
+ * Portions Copyrighted 2018 OGIS-RI Co., Ltd.
```

Comment:
I'm sorry I didn't notice immediately.
`2018` -> `2019` ?

Reply:
Thank you for pointing out. 2019 is correct.

Reply:
I'm sorry to bother you, but please check other files...

Reply:
Fixed all typos.

Reply:
OK, thanks!